### DISCUSSION TITLE
Function SelectedInput dengan Default All Options

### DISCUSSION THREAD

**Discussion by rulierinton** [teamalgoritma/community#73] [Q&A] [answered]
Bagaimana cara membuat default SelectedInput ALl options tidak hanya salah satu pilihan saja?
atau adakah alternatif lain untuk membuat filter yang pemilihan defaultnya "All"

Terimakasih 
![A](https://user-images.githubusercontent.com/101322908/165926763-084be9ee-d2de-40c5-bd3e-ccf6c7a84ac2.PNG)

**Comment by dwiknrd** [comment, marked as answer]
Halo [rulierinton](https://github.com/rulierinton)

`selectInput` merupakan salah satu jenis input yang bisa digunakan untuk memilih hanya satu value. Sehingga pada para parameter `selected` hanya dapat disetting dengan 1 value saja. 

Apabila ada kebutuhan untuk input yang dapat menampilkan lebih dari 1 value anda dapat menggunakan jenis input bernama `radioButtons`.

Untuk lebih lengkapnya anda dapat merujuk pada [cheatsheet shiny](https://shiny.rstudio.com/images/shiny-cheatsheet.pdf)

Semoga bermanfaat 😊
